utilities/decrypt_data/decrypt_multi.py

```
@@ -0,0 +1,237 @@
+        decrypted = {}
+        for item in result_list:
+            decrypted.update(item)
+        logger.info("Decrypted → %s", decrypted)
+        return decrypted
+
+    except requests.RequestException as e:
+        logger.error("Decrypt API error for %s: %s", to_send, e)
+        return {}
+
+
+def update_decrypted_values_for_table(table_config):
```

Ruff (0.12.2)

Missing return type annotation for public function `update_decrypted_values_for_table`

(ANN201)